utils/scrape.js

```diff
@@ -0,0 +1,102 @@
+import cheerio from 'cheerio';
```

Comment:
This file should live in `server/utils` since `cheerio` is not a dependency we want to have on the client.